vmas/scenarios/discovery.py

```diff
@@ -28,7 +28,17 @@ def make_world(self, batch_dim: int, device: torch.device, **kwargs):
         self._min_dist_between_entities = kwargs.pop("min_dist_between_entities", 0.2)
         self._lidar_range = kwargs.pop("lidar_range", 0.35)
         self._covering_range = kwargs.pop("covering_range", 0.25)
+
         self.use_agent_lidar = kwargs.pop("use_agent_lidar", False)
+        if self.use_agent_lidar:
+            self.n_lidar_rays_entities = kwargs.pop("n_lidar_rays_entities", 15)
+        else:
+            if "n_lidar_rays_entities" in kwargs:
+                raise ValueError(
+                    'Cannot set "n_lidar_rays_entities" when "use_agent_lidar" is False.'
+                )
+        self.n_lidar_rays_agents = kwargs.pop("n_lidar_rays_agents", 12)
```

Comment:
```suggestion
        self.n_lidar_rays_entities = kwargs.pop("n_lidar_rays_entities", 15)
        self.n_lidar_rays_agents = kwargs.pop("n_lidar_rays_agents", 12)
```

I was thinking about this and having the error would be a problem for BenchMARL since it cannot pass or not kwargs, I think the above suggestion will be enough

PS you had inverted entities and agents